habitat-lab/habitat/sims/habitat_simulator/debug_visualizer.py

```diff
@@ -77,6 +78,43 @@ def save(self, output_path: str, prefix: str = "") -> str:
         return file_path
 
 
+def draw_object_highlight(
```

Comment:
Can we control the thickness of the highlight as well? I imagine this can be important for some applications based on the camera resolution.

Reply:
I think this can be done in the debug_line_render object before it is passed into this function. https://aihabitat.org/docs/habitat-sim/habitat_sim.gfx.DebugLineRender.html#set_line_width 

So it shouldn't need to be explicitly controlled here. Rather the user application can set that for all debug renders.